spec.emu

```diff
@@ -177,7 +177,7 @@ contributors: Google, Ecma International
         1. Let _r_ be ResolveLocale(%DisplayNames%.[[AvailableLocales]], _requestedLocales_, _opt_, %DisplayNames%.[[RelevantExtensionKeys]]).
         1. Let _style_ be ? GetOption(_options_, `"style"`, `"string"`, &laquo; `"narrow"`, `"short"`, `"long"` &raquo;, `"long"`).
         1. Set _displayNames_.[[Style]] to _style_.
-        1. Let _type_ be ? GetOption(_options_, `"type"`, `"string"`, &laquo; `"language"`, `"region"`, `"script"`, `"currency"`, `"dateField"`, `"dateSymbol"` &raquo;, `"language"`).
+        1. Let _type_ be ? GetOption(_options_, `"type"`, `"string"`, &laquo; `"language"`, `"region"`, `"script"`, `"currency"`, `"date-field"`, `"date-symbol"` &raquo;, `"language"`).
```

Comment:
I think these should remain camelCase, per https://github.com/tc39/proposal-intl-displaynames/issues/32#issuecomment-503620717.

Reply:
@leobalter @sffc What is your view?

Reply:
we should avoid discuss this here until #32 is solved. Otherwise we have the same discussion in 2 different places.